core/src/avm2/globals.rs

```diff
@@ -743,5 +743,9 @@ fn load_playerglobal<'gc>(
 
     // Domain memory must be initialized after playerglobals is loaded because it relies on ByteArray.
     domain.init_default_domain_memory(activation)?;
```

Comment:
do we still need domain memory on the builtins' domain?

Reply:
We don't current use it, but I think we should keep it so that the playerglobals domain is co distent with other domains (all other domains will have that field initialized).

Reply:
OK 👍